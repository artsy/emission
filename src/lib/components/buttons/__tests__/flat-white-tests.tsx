import 'react-native'

import * as React from 'react'
import * as renderer from 'react-test-renderer'

import WhiteButton from '../flat_white'

it('renders properly', () => {
  const button = renderer.create(<WhiteButton selected={true} text={'I am a button'}/>).toJSON()
  expect(button).toMatchSnapshot()
})
