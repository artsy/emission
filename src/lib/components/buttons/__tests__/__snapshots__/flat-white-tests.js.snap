exports[`test renders properly 1`] = `
<TouchableHighlight
  activeOpacity={1}
  onPress={undefined}
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "rgba(0, 0, 0, 1)",
      "borderColor": "#e5e5e5",
      "borderWidth": 1,
      "justifyContent": "center",
    }
  }
  underlayColor="black">
  <View>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "fontSize": 12,
          },
          Object {
            "color": "black",
            "fontFamily": "AGaramondPro-Regular",
            "opacity": 1,
          },
          Object {
            "fontFamily": "Avant Garde Gothic ITCW01Dm",
          },
        ]
      }>
      I AM A BUTTON
    </Text>
  </View>
</TouchableHighlight>
`;
